install/kubernetes/Makefile

```diff
@@ -39,16 +44,29 @@ update-versions:
 		xargs -0 -l sed -i -e 's/'$(CILIUM_CHART_REGEX)'/\1 $(VERSION)/g'
 	@# Fix up the cilium tag
 	$(QUIET)for chart in $(CILIUM_VALUES); do							\
-		if echo $(VERSION) | grep -q $(LATEST_VERSION_REGEX); then				\
-			sed -i 's/'$(CILIUM_TAG_REGEX)'/\1 latest/' $$chart;				\
-			sed -i 's/'$(CILIUM_PULLPOLICY_REGEX)'/\1 Always/' $$chart;			\
-		elif echo $(VERSION) | grep -q $(DEV_VERSION_REGEX); then				\
-			sed -i 's/'$(CILIUM_TAG_REGEX)'/\1 v$(subst -dev,,$(VERSION))/' $$chart;	\
-			sed -i 's/'$(CILIUM_PULLPOLICY_REGEX)'/\1 Always/' $$chart;			\
-		else											\
-			sed -i 's/'$(CILIUM_TAG_REGEX)'/\1 v$(VERSION)/' $$chart;			\
-			sed -i 's/'$(CILIUM_PULLPOLICY_REGEX)'/\1 IfNotPresent/' $$chart;		\
-		fi; done
+		cilium_version="v$(VERSION)";								\
+		hubble_version=$(HUBBLE_UI_VERSION);							\
+		pull_policy="IfNotPresent";								\
+		if echo "$(VERSION)" | grep -q $(LATEST_VERSION_REGEX); then				\
+			cilium_version="latest";							\
+			hubble_version="latest";							\
+			pull_policy="Always";								\
+		elif echo "$(VERSION)" | grep -q $(DEV_VERSION_REGEX); then				\
+			cilium_version="v$(subst -dev,,$(VERSION))";					\
+			hubble_version="$(HUBBLE_UI_VERSION)";						\
+			pull_policy="Always";								\
+		fi;											\
+		for tag in $(CILIUM_TAGS); do								\
+			yq w -i $(CILIUM_VALUES) "$$tag" "$$cilium_version";				\
+		done;											\
+		for tag in $(HUBBLE_TAGS); do								\
+			yq w -i $(CILIUM_VALUES) "$$tag" "$$hubble_version";				\
+		done;											\
+		sed -i 's/'$(CILIUM_PULLPOLICY_REGEX)'/\1 '$$pull_policy'/' $$chart;			\
+	done
+	$(QUIET)yq w -i $(CILIUM_VALUES) etcd.image.tag $(MANAGED_ETCD_VERSION)
```

Comment:
This introduces `yq` as a dependency, maybe should consider adding something earlier in the target to check for its presence..? Wasn't sure the canonical way to add a CLI tool check.

Reply:
I couldn't find a canonical way or even another example in our code base, but maybe the following would do the trick:
```makefile
update-versions:
    @if ! yq --version &> /dev/null ; then echo "Please install yq"; exit 1; fi
```

Reply:
I can't find the package for my distribution that has `yq`, can we do this with a docker image? https://github.com/mikefarah/yq#run-with-docker

Reply:
A clean way to achieve that in make is to define a `.PHONY` target that check whether `jq` is installed and have it as a dependency, .e.g https://github.com/cilium/cilium/blob/v1.8/api/v1/Makefile#L51-L53.

Reply:
Cool, I'll keep that tucked behind my ear for next time it comes up.